debian/control

```diff
@@ -66,7 +67,7 @@ Architecture: linux-any
 Multi-Arch: same
 Depends: ${misc:Depends}, ${shlibs:Depends}
 Provides: libcxgb3-1,
-          libefa-1,
```

Comment:
There never was a libefa, so this shouldn't have been added, just delete it

Reply:
This package ships a `libefa.so.1` file. So providing libefa1 is nice to have.

Reply:
Auto provides should pick up the current Provides automatically.  Usually, you only have to give a manual Provides: tag for when you want an application linked against an older libibverbs and using an older lib<provider> to still pass package system dependency resolution.  Since there never was an older libefa for things to link against, there should be no requirement for a manual provides of an older libefa version (the current libefa should not be libefa-1).  But, that's all based on my knowledge of rpm as the package manager.  I can't speak for Debian, but if apt works similarly, then Jason would be right.

Reply:
The provides are only present to easy transition from the pre-rdma-core era. We don't need to add new provides for things that didn't exist prior to rdma-core.

Reply:
Debian does not have an "auto provides" functionality. You can remove the provided libefa1 if you like. It is only useful for users wanting to install that library and will not have an influence on the inter-package dependencies.